subt/main.py

```diff
@@ -459,7 +459,8 @@ def on_pose2d(self, timestamp, data):
         # pose3d
         dist3d = quaternion.rotate_vector([dist, 0, 0], self.orientation)
         self.xyz_quat = [a + b for a, b in zip(self.xyz_quat, dist3d)]
-        self.bus.publish('pose3d', [self.xyz_quat, self.orientation])
+        xyz_quat = [p+o for p, o in zip(self.xyz_quat, self.offset)]
```

Comment:
I would probably use space in `p + o` but ...

Reply:
I've added the spaces.